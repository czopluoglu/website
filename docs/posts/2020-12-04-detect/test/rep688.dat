00000101001101111111
01100001011100111111
11011101111011111011
10100000101110000111
11101101011111001110
01000001111011101100
10011001110010000000
00000001101011100100
11111001111101101001
10111111111010110111
11111111111011001110
11110111111110100101
11001010110100011111
11111111101000001011
10100001110000100001
10011001000010000010
00100100101011001110
10101100101100001001
11000100100010100001
01100011110010011110
01100001101010101011
11001101000000100100
11100101001111100111
00100101011011110111
11011111111111011111
11011000111111100110
11111101101110111111
10001010101010011011
11101000111100111010
01000000010001100110
11100100011101011011
00010100011110101001
00110001011000101011
11001100100000011010
10110110111111110110
10011110011111111111
00000001001010100111
10000000101011011111
11100000101110011011
01000101011011111000
11110111101100101110
00000100001000111111
10010100111010111111
00110001001010011111
10100001110000110100
00111011001110111011
00001101111000101111
11111101111100100111
10001111110010111111
11011011011100111111
00000000001000000110
11001001001000011011
10001010110110001110
00010111011010101100
11111111101110100010
10100101100001001011
10100110110010100001
00001000100001000100
01000001101001110110
00111001011011010010
00110000001000000111
11111001111110101111
10101101111011011110
11000101101010110101
00101001101011011011
10001001011110101110
10011100101001101111
11011001000100100010
11101111111101100111
01100101101101111111
10101011011010101110
01110000001100111011
11100100111101111101
10100101011011110010
01001101001110110010
00011101101111110100
11111101010110100010
10001001100101001111
11001001000100110000
11101011001011110111
10001100111111000110
10110010100000010000
00001001110100101101
10011011010001111110
01000001001111111101
01101001101010000000
00100010101110100111
01110001100000101010
01100011111110001010
10000001001101111000
10101000010110011010
10100100001111111111
00100111111010110111
11100011100101101111
01000111111000000111
00001000111010001011
11110101111010001000
11011111110011000000
10001111100110100010
10000111001001011011
01110111101010010010
00100000001011000111
11110100001001011101
11100111110011000001
00011101010011000010
01000001011111111111
11101111101111000100
00000011111101111010
10111101111101010111
00110111101101000110
00100110001110101001
11111100100010100001
10011011111111111010
10000100100100111011
11101000011110101110
11101111110111001011
00110001011011001011
10111000010000101111
10000110010011010011
10000001111111110010
10011010011010100110
11010110011110010111
10101001111110001100
10101111011001001111
10000101001100110110
11110100100110111111
01001001111110111111
00101000111110011110
10001001011110111111
00110101111011101101
10000001001010111110
11010000010011100101
11001101001110100011
11000111110010101110
11111010011110000001
11111111111111001110
01011001110111110111
11001101011000000001
10001101011010111001
00111101101101000000
10110101001100110111
10010001100100010111
11111111110110101010
10111111111111111110
10011111110110000010
00100101111010111110
10100110111000010011
11111001011101000101
11000101110100100010
10010111110110000110
10001111001111111011
01000000011101000010
10000101010010110010
00111010111000111011
10110110110010101000
01110110111110101101
11001110100010110011
00000000101000100110
10011000011000111010
00000000000000111001
11001001111101100011
11100001010100101110
01110000111010000110
10000000001110001010
11110101011011111110
11100000000111100110
11111111011111100111
11101111101110100110
01111001000110100101
01010000111111111011
10000000001000000001
11100101000111010010
11110110111011111100
00000001001001111100
11101101010111111111
11010000001010101001
00001001110100011011
10100101101111011111
00000000001000011011
11110101011001100001
11110011111110100010
11011100010100000011
11000011111110101011
10011111110111100011
00011100000010100010
11010111101000010110
00000001110010101110
10110111111001001010
10001101111011101011
11101000111111100110
11000010011100101011
00101111111111001111
11111111111001010001
01000111011101111010
11110011000010111011
00001111111101101011
01100111111010001111
00110001001100001001
10001100001110101010
01101001111110000000
11110101010010000110
01100101100110111011
11111101001010100001
00101011111010100000
11100100100101011111
11101011111100110110
00111101010000000000
00101001110001011110
10101001011111111100
10000111011000100111
00001011111101101110
10010110101010011011
10001101011101100010
10111111111111111111
11010111111110101100
00100000001001101110
11001000011000011111
00110000010100110111
01110100111000010000
10101001110000001111
00100001001101111111
00010101110110101100
11101111001000001010
00110000001110001110
11111111110010111111
11001111111100010111
01000000010110000111
11010101111001100111
00110101111110101111
10001101000100101011
10011011110000001010
01110101100011000110
00111011001011011011
11000000011001000010
11111001011010110111
11110101111010101111
11011010111010101111
10111101111011100100
11101010010010110011
01110000001110110111
01000110001010101111
10111011011111101011
10011011101111001111
00000000001100110111
01111111111110011011
00001100001001111111
01011111011100100110
11010001110011100010
01000000100011000111
11111111110010101111
