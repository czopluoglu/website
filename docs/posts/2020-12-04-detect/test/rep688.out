-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep688.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.50
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  2
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   1.0199
                   IDN index value:   0.7684
                           Ratio r:   0.7151

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    1    1    1    1    1    1    1    1
    2    2    1    2    2    2    2    2    2    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    3    4    5    6    7    8    9   10
   13

   -----------CLUSTER  2-------------

   11   12   14   15   16   17   18   19   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+-++++--+----------
   +d++-+--++--+-+----+
   -+d+-+-++-++-----+-+
   +++d+++++++---------
   +--+d-++++--+-----+-
   ++++-d+++-+--+-----+
   +--+++d++++-------+-
   --+++++d++---++-----
   -+++++++d++------+--
   ++-++-+++d+---+--+--
   --++-++-++d---+-----
   --+--------d++++++-+
   -+--+------+d++-+++-
   -----+-+---++d-+-+--
   -+-----+-++++-d+-+--
   -----------+-++d++++
   -----------++--+d-++
   --+-----++-+++++-d+-
   ----+-+-----+--+++d+
   -++--+-----+---++-+d
 
 No cross validation for this DETECT run
